phpunit.xml.dist

```diff
@@ -26,6 +26,7 @@
             <directory>./src/Symfony/Bridge/*/Tests/</directory>
             <directory>./src/Symfony/Component/*/Tests/</directory>
             <directory>./src/Symfony/Component/*/*/Tests/</directory>
+            <directory>./src/Symfony/Contract/*/Tests/</directory>
             <directory>./src/Symfony/Bundle/*/Tests/</directory>
```

Comment:
Is there a specific reason why these rules aren't sorted?

Reply:
No specific one, just history (we don't reorder to limit useless merge conflicts.)